### DISCUSSION TITLE
Pre-Conference

### DISCUSSION THREAD

**Discussion by sqlworldwide** [dataplat/DataSaturdays#159] [Q&A]
Most likely part of the session design. Not sure if any different needs to happen other than adding a full day category. Might also need the option to chose a different day. Most pre-conference happen on Friday.

**Comment by SQLClause** [comment]
1) Pre-cons are paid events.
2) Sessionize is only free for free events
3) We can of course deliver reccomendations on solutions such as Office Forms or Google forms for pre-cons

**Comment by SQLDBAWithABeard** [comment]
that is true but Sessionize pricing is 499 for a single event, we may be able to do some negotiate

**Comment by SQLClause** [comment]
Yes, I suspect it's possible to negotiate a common license. We just have to keep this limitation in mind :)
